### PR TITLE
Update base.po

### PR DESCRIPTION
**Pull request by celikj**
corrected the meaning of "locale"' fixed some typos and added "Disable Copy-on-Write" and "Use compression"

### CODE REVIEW

File: archinstall/locales/tr/LC_MESSAGES/base.po [hunk 1]
```diff
@@ -684,7 +679,7 @@ msgid "Select one or more hard drives to use and configure\n"
 msgstr "Kullanmak ve yapılandırmak için bir ya da daha fazla sabit disk seçin\n"
 
 msgid "Any modifications to the existing setting will reset the disk layout!"
-msgstr "Mevcut ayara herhangi bir modifikasyon disk şemasını sıfırlayacaktır!"
+msgstr "Mevcut ayarda yapılacak herhangi bir değişiklik disk düzenini sıfırlayacaktır!!"
```
Comment (celikj) [inline archinstall/locales/tr/LC_MESSAGES/base.po:682]: there is doube !. sorry for mistake

**Review by celikj — COMMENTED** [review]
sorry for doube !

**Comment by svartkanin** [comment]
@celikj could you rerun the script again please to generate the .mo file as well and commit that

**Comment by celikj** [comment]
@svartkanin done

**Review by svartkanin — APPROVED** [review]

**Pull request merged**